docs/pages/api-docs/calendar-picker-skeleton.json

```diff
@@ -1,17 +1,8 @@
 {
-  "props": {},
+  "props": { "classes": { "type": { "name": "object" } }, "sx": { "type": { "name": "object" } } },
   "name": "CalendarPickerSkeleton",
   "styles": {
-    "classes": [
-      "hidden",
-      "root",
-      "loadingContainer",
-      "weekContainer",
-      "week",
-      "daysHeader",
-      "weekDayLabel",
-      "daySkeleton"
-    ],
```

Comment:
These are `classes` defined in another component `PickersCalendar`, the changed is correct.

Reply:
👍 great!